client/js/screens/Campaigns/styles.js

```diff
@@ -1,3 +1,88 @@
 import {StyleSheet} from 'react-native';
-const styles = StyleSheet.create({});
+const styles = StyleSheet.create({
+  xxxIcon: {
```

Comment:
can you please explain this name? I am not sure what it is for.

Reply:
It is a group of x's in a png, xGroupIcon could be a less lurid name

Reply:
Done